lib/jekyll-relative-links/generator.rb

```diff
@@ -65,7 +65,9 @@ def url_for_path(path)
       extension = File.extname(path)
       return unless markdown_extension?(extension)
 
-      page = site.pages.find { |p| p.path == path }
+      page = site.pages.find { |p| p.path == path } ||
+        site.docs_to_write.find { |p| p.relative_path == path } ||
+        site.static_files.find { |p| p.relative_path == path }
```

Comment:
I ended up removing static files for now. Since this only supports markdown links and not `![]` image references it's not as useful. So it can be added in another PR if it's useful for another case.